tests/include/boost/ut.hpp

```diff
@@ -331,28 +331,28 @@ struct identity {
 };
 
 template <class T>
-struct function_traits : function_traits<decltype(&T::operator())> {};
+struct function_trait : function_trait<decltype(&T::operator())> {};
```

Comment:
BoostExt.ut is tested under gcc 9 and clang 9. What's the problem with this name?

Reply:
Sorry, that was a global search/replace error. I reverted that change.